packages/common/src/types/slackSettings.ts

```diff
@@ -5,12 +5,14 @@ export type SlackSettings = {
     token?: string;
     scopes: string[];
     notificationChannel: string | undefined;
+    appProfilePhotoUrl: string | undefined;
 };
 
 export const slackRequiredScopes = [
     'links:read',
     'links:write',
     'chat:write',
+    'chat:write.customize',
```

Comment:
we have to update all apps right?

Reply:
Yep yep, we will need to update all apps 👌